build.zig.zon

```diff
@@ -1,6 +1,12 @@
 .{
-    .name = "go",
+    .name = .go,
     .version = "0.0.0",
-    .dependencies = .{},
+    .dependencies = .{
```

Comment:
Is this here by mistake? I'm not sure why it depends on itself (and either way, this is a path to your local dev machine)